definitions/infra-host/relationships.yml

```diff
@@ -0,0 +1,11 @@
+  #  SELECT entityGuid (target), nodeName (source) FRom Span where nodeName is not null
+  #  existing relationship
+relationships:
```

Comment:
should this one be on in ext-service. If i'm not wrong the nodeName is reported by the service right?

Reply:
well.. problem is if we do that, we would need to declare this rule in N places.

Reply:
yeah but I think you can't avoid it

Declaring it here implies that on the host telemetry you expect datapoints to contain attributes for *every* service that runs in that node. Hence, you have to alter the agent to pull those services and report them in separate attributes.  Which I think has obvious complication (e.g. if you have 2 services running in a host, the agent needs 2 attributes, and we need 2 rules to understand each).

I agree with @naxhh that this rule makes more sense on the service, as given a datapoint, there will always be a 1-1 relationship to a node (a single attribute).  If we have another type of entity that also can bring the nodeName, then yes, we need a rule there.

Reply:
Considering this, we could likely remove the `source` node. No need for it if it's to always be the `entityGuid` of the defining entity type